graphene_sqlalchemy/types.py

```diff
@@ -207,10 +207,10 @@ def __init_subclass_with_meta__(
         _meta=None,
         **options
     ):
-        assert is_mapped_class(model), (
-            "You need to pass a valid SQLAlchemy Model in " '{}.Meta, received "{}".'
-        ).format(cls.__name__, model)
-
+        if not is_mapped_class(model):
```

Comment:
`assert` is only evaluated when the interpreter is running in "debug" mode and is ignored when optimizations are enabled with the `-O` command line switch. I personally think it's proper to have this check every time and what you have written is an improvement, but to match the previous behavior it might be best to wrap this in an `if __debug__:` check.

https://docs.python.org/3/reference/simple_stmts.html#grammar-token-assert-stmt

Perhaps
```
if __debug__ and not is_mapped_class(model):
    raise ValueError(...)
```